Documentation/Books/Manual/Programs/Arangorestore/ParallelRestore.md

```diff
@@ -1,2 +1,198 @@
-Parallel Restore Procudure
+Parallel Restore Procedure
 ==========================
+
+The following parallel restore procedure is recommended to speed up the
+performance of [_arangorestore_](../Arangorestore/README.md) in a
+cluster environment.
```

Comment:
Misses a description of how this is achieved. It may not be clear at this point.

As far as I understand, you restore the data on each coordinator individually without replication (what about sharding?), then re-enable replication - it should not be necessary to replicate the data over the network, because it was already restored locally right?

What if you have 3 nodes, but a replication factor of 2 however? Don't you restore the data unnecessarily on one of the nodes? Or is it rejected by the coordinator before the data gets sent to the dbserver?

What if coordinator and dbserver are not on the same physical machine? Will it still be significantly faster to do a parallel restore? (The data needs to be sent over the network anyway in this case, but it might still be more efficient than the ArangoDB replication.)

Reply:
This is good feedback. More context and explanations are definitely needed. I have pushed something now. Please review, can it be improved more? Thanks,

Reply:
i have added a description - resolving. the imprevements are due to: 1 using replication factor 1, and 2 restore multiple collections at the same time from different coordinators (vs restoring from a single coordinator)